vc_assistance/indico_vc_assistance/client/style.scss

```diff
@@ -0,0 +1,22 @@
+// This file is part of the CERN Indico plugins.
+// Copyright (C) 2014 - 2018 CERN
+//
+// The CERN Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License; see
+// the LICENSE file for more details.
+@import 'base/palette';
+
+.overview-results {
+
```

Comment:
shouldn't have an empty line here (but there should be one after the file header)